modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -4058,6 +4058,7 @@ content-related-to-x=Content Related to <em>{0}</em>
 content-root-prefix=Content Root Prefix
 content-security-policy=Content Security Policy
 content-security-policy-configuration-name=Content Security Policy Configuration
+content-security-policy-excluded-uri-path-help=Request URI paths starting with the following string will not be modified by Content Security Policy. 
```

Comment:
This sounds like it is the Request URIs that get modified. Maybe It could read...?

> Resources at paths starting with the string will not be subject to the Content Security Policy.